fixed look and feels of genome widget a bit

nathandunn opened the pull request:
Fixes AGR-310.

nathandunn commented:
Let me know if you have any UI recommendations for those buttons.

<img width="643" alt="screen shot 2017-06-05 at 10 35 23 am" src="https://cloud.githubusercontent.com/assets/751274/26795495/b4febb10-49da-11e7-97bb-4cdef7c78e83.png">

paaatrick commented:
I was going to ask what the purpose of those links/buttons is in the first place? The first one takes you to the same place as just clicking the image. And the second one just gives you the same (maybe slightly wider) image again? I guess my point is the first one seems redundant and I'm not sure how valuable the second one is. 

But if you think they're important to keep, yeah, I would tone it down a bit. I think `btn-secondary` would be more appropriate than `btn-warning`. I'd take off the icons, too. Maybe even move them below the image, since they seem of secondary importance to the image itself?

Regarding the section name change, someone else requested it be called "Gene Model" (https://agr-jira.atlassian.net/browse/AGR-436)

paaatrick commented on webapp/src/containers/genePage/transcriptInlineViewer.js:
```diff
@@ -46,8 +46,10 @@ class TranscriptViewer extends Component {
-        <a href={externalJbrowseUrl}>Overview<i className="fa fa-link" /> </a>
+        <a href={externalJbrowseUrl.replace('overview.html','index.html')} className="btn btn-warning btn-sm">Browse <i className="fa fa-search" /></a>
+        &nbsp;
+        &nbsp;
```
Are these just to add space between the buttons? Can that be done with CSS instead if needed?

nathandunn commented:
Fixed:
- changed it to "Gene Mode" for https://agr-jira.atlassian.net/browse/AGR-436
- made secondary buttons (for some reason btn-default wasn't working), got rid of ```&nbsp;```

I left the buttons on there even though the functionality is duplicate.  Mostly because there really aren't any directions associated with it and the image itself doesn't imply a link for the user.   I tried a blue background for the image, but of course that looks horrible as well.   When we move to divs, I think that will help.

nathandunn commented:
Oh, I removed the snapshot button.   It was mostly there for dev purposes.
